modules/cdn/profile/.test/afd/main.test.bicep

```diff
@@ -1,132 +1,132 @@
-                  redirectType: 'PermanentRedirect'
-                  destinationProtocol: 'Https'
-                  customPath: '/test123'
-                  customHostname: 'dev-etradefd.trade.azure.defra.cloud'
-                }
-              }
-            ]
-          }
-        ]
-      }
-    ]
-    afdEndpoints: [
-      {
-        name: 'dep-${namePrefix}-test-${serviceShort}-afd-endpoint'
-        routes: [
-          {
-            name: 'dep-${namePrefix}-test-${serviceShort}-afd-route'
-            originGroupName: 'dep-${namePrefix}-test-${serviceShort}-origin-group'
-            customDomainName: 'dep-${namePrefix}-test-${serviceShort}-custom-domain'
-            ruleSets: [
-              {
-                name: 'dep${namePrefix}test${serviceShort}ruleset'
-              }
-            ]
-          }
-        ]
-      }
-    ]
-  }
-}
-
+targetScope = 'subscription'
+
+// ========== //
+// Parameters //
+// ========== //
+
+@description('Optional. The name of the resource group to deploy for testing purposes.')
+@maxLength(90)
+param resourceGroupName string = 'dep-${namePrefix}-cdn.profiles-${serviceShort}-rg'
+
+@description('Optional. The location to deploy resources to.')
+param location string = deployment().location
+
+@description('Optional. A short identifier for the kind of deployment. Should be kept short to not run into resource-name length-constraints.')
+param serviceShort string = 'cdnpafd'
+
+@description('Optional. Enable telemetry via a Globally Unique Identifier (GUID).')
+param enableDefaultTelemetry bool = true
+
+@description('Optional. A token to inject into the name of each resource.')
+param namePrefix string = '[[namePrefix]]'
+
+// ============ //
+// Dependencies //
+// ============ //
+
+// General resources
+// =================
+resource resourceGroup 'Microsoft.Resources/resourceGroups@2021-04-01' = {
+  name: resourceGroupName
+  location: location
+}
+
+module nestedDependencies 'dependencies.bicep' = {
+  scope: resourceGroup
+  name: '${uniqueString(deployment().name, location)}-nestedDependencies'
+  params: {
+    storageAccountName: 'dep${namePrefix}cdnstore${serviceShort}'
+    managedIdentityName: 'dep-${namePrefix}-msi-${serviceShort}'
+  }
+}
+
+// ============== //
+// Test Execution //
+// ============== //
+
+module testDeployment '../../main.bicep' = {
+  scope: resourceGroup
+  name: '${uniqueString(deployment().name, location)}-test-${serviceShort}'
+  params: {
+    name: 'dep-${namePrefix}-test-${serviceShort}'
+    location: 'global'
+    lock: {
+      kind: 'CanNotDelete'
+      name: 'myCustomLockName'
+    }
+    originResponseTimeoutSeconds: 60
+    sku: 'Standard_AzureFrontDoor'
+    enableDefaultTelemetry: enableDefaultTelemetry
+    roleAssignments: [
+      {
+        roleDefinitionIdOrName: 'Reader'
+        principalId: nestedDependencies.outputs.managedIdentityPrincipalId
+
+        principalType: 'ServicePrincipal'
```

Comment:
```suggestion
        principalId: nestedDependencies.outputs.managedIdentityPrincipalId
        principalType: 'ServicePrincipal'
```

Reply:
Please bluk edit this, as it affects several files. Not sure why it shows in your PR